### PR TITLE
Update input_controller.js

### PR DESCRIPTION
**Pull request by algha**
if mask is null or empty, cant not type japanese character.
in this edit, solved typing japanese character.

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by tabuna** [comment]
I tried to reproduce, but it does not work for me. Any language was inserted without any problems. 
But I wrote only in English and Russian, the rest was inserted by copying. 
Does this happen at the moment of pressing?
Снимок экрана от 2019-04-17 10 40 28

**Comment by tabuna** [comment]
I understand that it may be difficult to explain and reproduce.
 But the change is not big, let's take it

File: resources/js/controllers/fields/input_controller.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ export default class extends Controller {
         } catch (e) {
             // default
         }
-
-        Inputmask(mask).mask(element);
+        if (mask != '') {
```
Comment (tabuna) [inline resources/js/controllers/fields/input_controller.js]: As far as I remember, the empty string returns `false`.
```php
if (mask) {
    Inputmask(mask).mask(element);
}
```

Comment (algha) [reply resources/js/controllers/fields/input_controller.js]: ok, thanks.

**Pull request merged**